src/platform_impl/linux/window.rs

```diff
@@ -101,17 +107,20 @@ impl Window {
       .insert(window_id);
 
     // Set Width/Height & Resizable
-    let win_scale_factor = window.scale_factor();
     let (width, height) = attributes
       .inner_size
       .map(|size| size.to_logical::<f64>(win_scale_factor as f64).into())
       .unwrap_or((800, 600));
-    window.set_default_size(1, 1);
-    window.resize(width, height);
+
+    let window_clone = window.clone();
+    glib::idle_add_local_once(move || {
+      window_clone.set_resizable(true);
+      window_clone.resize(width, height);
+    });
```

Comment:
when resizable is set to false in the dev branch, a window with the size set to `220 x 90` is created as `332 x 200` without generating the inner size. 
However, if set_resizable(true) is added before resizing in the current branch, the window is created with the specified size.

## In dev branch
<img width="945" alt="스크린샷 2024-10-02 오후 5 43 23" src="https://github.com/user-attachments/assets/06ff40da-35af-45ae-8129-c2465e0842e1">

## Before adding `set_resizable(true)`

<img width="945" alt="스크린샷 2024-10-02 오후 5 48 47" src="https://github.com/user-attachments/assets/c36018ec-7be6-4ae9-b1b9-38cb4085c3b3">

## After adding `set_resizable(true)`

<img width="945" alt="스크린샷 2024-10-02 오후 6 07 53" src="https://github.com/user-attachments/assets/1ea8f7eb-7045-4863-8a92-1096e080d1f9">